selfdrive/sensord/rawgps/rawgpsd.py

```diff
@@ -112,7 +120,11 @@ def setup_quectel(diag: ModemDiag):
   mmcli("--command='AT+QGPSCFG=\"dpoenable\",0'")
   # don't automatically turn on GNSS on powerup
   mmcli("--command='AT+QGPSCFG=\"autogps\",0'")
-  mmcli("--location-enable-gps-raw --location-enable-gps-nmea")
+  mmcli("--command='AT+QGPSSUPLURL=\"supl.google.com:7275\"'")
```

Comment:
if we're only doing AT commands now, let's change `mmcli` to take an AT command directly and rename to `at_cmd`